#!/usr/bin/env python
PACKAGE = "ibvs_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("p", double_t, 0, "", 0.4, 0, 0.5)

gen.add("k1", double_t, 0, "", 10, 0, 500)
gen.add("k2", double_t, 0, "", 100, 0, 500)
gen.add("k3", double_t, 0, "", 0.05, 0, 10)
gen.add("k4", double_t, 0, "", 0.05, 0, 10)
gen.add("k_error_z", double_t, 0, "", 1, 1, 100)
gen.add("k_error_y", double_t, 0, "", 1, 1, 100)

gen.add("t_E_T_x",double_t, 0, "", -0.06, -0.5,0.5)
gen.add("error_t_z", double_t, 0, "", 0.03, 0, 0.1)

exit(gen.generate(PACKAGE, "param", "Param"))